### PR TITLE
New package: go-md2man

### PR DESCRIPTION
**Pull request by DarylGrunau**
This PR provides support to build/install go-md2man, a markdown 2 man converter.

### CODE REVIEW

File: var/spack/repos/builtin/packages/go-md2man/package.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    version('1.0.10', sha256='76aa56849123b99b95fcea2b15502fd886dead9a5c35be7f78bdc2bad6be8d99')
+
+    depends_on('go')
+
+    resource(name='blackfriday',
+             url='https://github.com/russross/blackfriday/archive/v1.5.2.tar.gz',
+             sha256='626138a08abb8579474a555e9d45cb5260629a2c07e8834428620a650dc9f195',
+             placement='blackfriday',
+             destination='{0}'.format(join_path(
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/go-md2man/package.py]: No need for `'{0}'.format()` here, `join_path` should already return a string-like thing.

File: var/spack/repos/builtin/packages/go-md2man/package.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+               'src', 'github.com', 'cpuguy83'))
+
+        ln = which('ln')
+        ln('-s', self.stage.source_path, join_path(
+           'src', 'github.com', 'cpuguy83', 'go-md2man'))
+
+    def install(self, spec, prefix):
+
+        with working_dir('src'):
+            env['GOPATH'] = '{0}'.format(self.stage.source_path)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/go-md2man/package.py]: No need for `'{0}'.format()` here either

File: var/spack/repos/builtin/packages/go-md2man/package.py (hunk 1 above)
Comment (DarylGrunau) [reply var/spack/repos/builtin/packages/go-md2man/package.py]: Ok

File: var/spack/repos/builtin/packages/go-md2man/package.py (hunk 2 above)
Comment (DarylGrunau) [reply var/spack/repos/builtin/packages/go-md2man/package.py]: See new content ...

**Pull request merged**